resources/js/main.js

```diff
@@ -0,0 +1,60 @@
+
+const ARRAY_LENGTH = 5;
+const MAX_INT = 100;
+
+console.log('START: Generate Random Array');
+const randomNumbers = generateRandomArray(ARRAY_LENGTH, MAX_INT);
```

Comment:
Because I said `random numbers` did you feel like a random number generator was necessary?

Reply:
I interpreted "5 random numbers" as meant to use `Math.random()`. If it was just "5 numbers" instead, I likely would've thrown 5 numbers directly into an array.